frame/support/src/traits/try_runtime.rs

```diff
@@ -34,6 +34,10 @@ pub enum Select {
 	///
 	/// Pallet names are obtained from [`super::PalletInfoAccess`].
 	Only(Vec<Vec<u8>>),
+	/// Run only fast running tests.
```

Comment:
This will run all pallets, but in fast mode right? Or else I am not sure how it work with that regards.

Reply:
Yes, it is same as `All` but the pallets can choose to ignore some tests in the fast mode.